[Azure AD] [Governance] updated properties table and changelog (#14272)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/assignmentreviewsettings.md

| Property                     | Type                      | Description |
| :--------------------------- | :------------------------ | :---------- |

> | accessReviewTimeoutBehavior | microsoft.graph.accessReviewTimeoutBehavior | Default decision result if request is not reviewed within the time period specified in `durationInDays`. |

Author: The type "microsoft.graph.accessreviewtimeoutbehavior" is causing an error.

> | accessReviewTimeoutBehavior | microsoft.graph.accessReviewTimeoutBehavior | Default decision result if request is not reviewed within the time period specified in `durationInDays`. |
> | durationInDays | Int32 | The number of days to allow input from reviewers.|
> | isAccessRecommendationEnabled | boolean | Display recommendation to reviewer or not. The default value is true |
> | isApprovalJustificationRequired | boolean | Require justification for reviewer or not. The default value is true. |

Reviewer: ```suggestion
| accessReviewTimeoutBehavior | accessReviewTimeoutBehavior | The default decision to apply if the request is not reviewed within the period specified in **durationInDays**. |
| durationInDays | Int32 | The number of days within which reviewers should provide input.|
| isAccessRecommendationEnabled | Boolean | Specifies whether to display recommendations to the reviewer. The default value is `true` |
| isApprovalJustificationRequired | Boolean | Specifies whether the reviewer must provide justification for the approval. The default value is `true`. |
```

| isEnabled| Boolean | If true, access reviews are required for assignments from this policy. |
| recurrenceType | String | The interval for recurrence, such as `monthly` or `quarterly`. |
| reviewerType | String | Who should be asked to do the review, either `Self` or `Reviewers`. |
| reviewers | [userSet](userset.md) collection | If the reviewerType is `Reviewers`, this collection specifies the users who will be reviewers, either by ID or as members of a group, using a collection of [singleUser](singleuser.md) and [groupMembers](groupmembers.md). |
| startDateTime | DateTimeOffset | When the first review should start. |

> 

Reviewer: Using [Option 2 of documenting Enums](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103158/Enumerations?anchor=second-option%3A-in-parent-resource-topic), you'll add a H3 section just after the properties table.

> ## JSON representation

Reviewer: Add the new properties to the JSON representation.

| Property                     | Type                      | Description |
| :--------------------------- | :------------------------ | :---------- |

> | accessReviewTimeoutBehavior | accessReviewTimeoutBehavior | The default decision to apply if the request is not reviewed within the period specified in **durationInDays**. |

Reviewer: Applying the Enum guidance
```suggestion
| accessReviewTimeoutBehavior | [accessReviewTimeoutBehavior](#accessreviewtimeoutbehavior-values) | The default decision to apply if the request is not reviewed within the period specified in **durationInDays**. The possible values are: `acceptAccessRecommendation`, `keepAccess`, `removeAccess`, and `unknownFutureValue`. |
```

| durationInDays | Int32 | The number of days within which reviewers should provide input.|
| isAccessRecommendationEnabled | Boolean | Specifies whether to display recommendations to the reviewer. The default value is `true` |
| isApprovalJustificationRequired | Boolean | Specifies whether the reviewer must provide justification for the approval. The default value is `true`. |
| isEnabled| Boolean | If true, access reviews are required for assignments from this policy. |
| recurrenceType | String | The interval for recurrence, such as `monthly` or `quarterly`. |
| reviewerType | String | Who should be asked to do the review, either `Self` or `Reviewers`. |
| reviewers | [userSet](userset.md) collection | If the reviewerType is `Reviewers`, this collection specifies the users who will be reviewers, either by ID or as members of a group, using a collection of [singleUser](singleuser.md) and [groupMembers](groupmembers.md). |
| startDateTime | DateTimeOffset | When the first review should start. |

> ### Enum Type

Reviewer: Applying the guidance
```suggestion
### accessReviewTimeoutBehavior values
```

| Member | Description |
|:---------------|:--------|:----------|

> | acceptAccessRecommendation | Review decision to take recommendations from access review to accept/remove access to access package. The general rule for AR recommendations is if last user sign in more than 30 days, it is recommended to remove access to that user. |
> | keepAccess | Review decision is to keep current access. |
> | removeAccess | Review decision is to remove access to access package. |
> | unknownFutureValue | Sentinel member for evolvable enum |

Reviewer: ```suggestion
| acceptAccessRecommendation | The review decision to take recommendations from access review to accept/remove access to access package. The general rule for AR recommendations is if last user sign in more than 30 days, it is recommended to remove access to that user. |
| keepAccess | The review decision is to keep current access. |
| removeAccess | The review decision is to remove access to access package. |
| unknownFutureValue | Evolvable enumeration sentinel value. Do not use. |
```
